factorio/Moon_Logic/control.lua

```diff
@@ -262,17 +275,13 @@ local function mlc_update_output(mlc, output_raw)
 	for _, k in ipairs{'red', 'green'} do
 		ps, ecc = {}, mlc['out_'..k].get_or_create_control_behavior()
 		if not (ecc and ecc.valid) then goto skip end
-		n, n_max = 1, ecc.signals_count
+		n, n_max = 1, 1
```

Comment:
This needs to be fixed. AFAIK, this max is no longer applicable.

From what I understand (which is not much lol), this would limit the amount of output signals to just 1, which is inconsistent with the new constant combinator, which can output all signals. I think, if you remove the n_max and the n_max check, this should work as originally intended.

Reply:
I believe this was originally designed to be consistent with the old constant combinator behavior

Reply:
i forgot to remove the variable and they completely removed the max number of signals per constant combinator

Reply:
I read the diff wrong my bad. Have you tested it? Does it work or are you getting another error lol? If so, open another issue, and you can start work on another PR for it.

Reply:
Bit busy working on a L4D2 server rn

Reply:
As far as testing goes, I didn't find any other problems.